pkgs/apps/gamess-us/default.nix

```diff
@@ -121,6 +124,7 @@ stdenv.mkDerivation rec {
 
     # Copy the interesting scripts and executables
     cp gamess.${version}.x rungms $out/bin/.
+    cp $(find -name ddikick.x) $out/bin/.
```

Comment:
This seems to fail:
`cp: missing destination file operand after '/nix/store/9pys33yphx6nsqx9qlfxrcwhf1yjx064-gamess-us-2021R2P1/bin/.'`

Reply:
You are right, the documentation actually explicitly states that ddikick.x is not created if MPI is used ... Should be fixed now.